person.rb

```diff
@@ -13,6 +13,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @books_rented = []
```

Comment:
- [ ] `@rentals` or in this case `@books_rented` should be added to the attribute_accessor.